docs/install_ovos_core.md

```diff
@@ -1,70 +1,185 @@
 
-### from source
+**alpha version**
 
-We suggest you do this in a virtualenv:
+`pip install --pre ovos-core[mycroft]`
 
-`pip install git+https://github.com/OpenVoiceOS/ovos-core[all]`
+**latest stable**
 
-### from pypi
+ovos-core 0.0.7 does not include the new `extras [mycroft]` so we use `[all]`
 
 `pip install ovos-core[all]`
 
-## Running ovos-core
+This should install everything needed for a basic OVOS software stack
+
+There are additional `extras` options available other than `[mycroft]` and can be found in the [ovos-core setup.py](https://github.com/OpenVoiceOS/ovos-core/blob/dev/setup.py) file.
+
+[Starting OVOS](starting_intro.md)
+
+### Individual Modules
+Each module can be installed independently to only inclue the parts needed or wanted for a specific system.
+
+**ovos-core**
+
+`pip install --pre ovos-core`
+
+[Starting Modules - core](starting_modules.md#core)
+
+**ovos-messagebus**
+
+`pip install --pre ovos-messagebus`
+
+[Starting Modules - messagebus](starting_modules.md#messagebus)
+
+**ovos-audio**
+
+`pip install --pre ovos-audio`
+
+[Starting Modules - audio](starting_modules.md#audio)
+
+**dinkum-listener**
+
+`pip install --pre ovos-dinkum-listener`
+
+[Starting Modules - listener](starting_modules.md#listener)
+
+**ovos-phal**
+
+`pip install --pre ovos-phal`
+
+[Starting Modules - phal](starting_modules.md#phal)
+
+## From Source
+We will use git to clone the repositories to a local directory.  While not specificly necessasary, we are assuming this to be the users `HOME` directory.
+
+### ovos-core
+Install `ovos-core` from github source files
+
+`git clone https://github.com/OpenVoiceOS/ovos-core`
+
+The ovos-core repository provides `extra` requirements files.  For the complete stack, we will use the `mycroft.txt` file.
+
+`pip install ~/ovos-core[mycroft]`
+
+This should install everything needed to use the basic OVOS software stack.
+
+**NOTE** this also installs `lgpl` licenced software
+
+[Starting OVOS](starting_intro.md)
+
+### Install individual modules
+
+Some systems may not require a full install of OVOS.  Luckly, it can be installed as individual modules.
+
+**core library**
+
+`git clone https://github.com/OpenVoiceOS/ovos-core`
+
+`pip install ~/ovos-core`
+
+This is the minimal library needed as the brain of the system.  There are no skills, no messagebus, and no plugins installed yet
+
+[Starting Core](starting_modules.md#core)
+
+**messagebus**
+
+`git clone https://github.com/OpenVoiceOS/ovos-messagebus`
+
+`pip install ~/ovos-messagebus`
+
+This is the nervous system of OVOS needed for modules to talk to eachother
+
+[Starting the Messagebus](starting_modules.md#messagebus)
+
+**listener**
+
+OVOS has updated thier listener to use `ovos-dinkum-listener` instead of `ovos-listener`.  It is code from [mycroft-dinkum](https://github.com/MycroftAI/mycroft-dinkum) adopted for use with the OVOS ecosystem.  Previous listeners are still available, but not recommended.
```

Comment:
```suggestion
OVOS has updated thier listener to use `ovos-dinkum-listener` instead of `ovos-listener`. It is code from [mycroft-dinkum](https://github.com/MycroftAI/mycroft-dinkum) adopted for use with the OVOS ecosystem.  Previous listeners are still available, but not recommended.
```